src/store/flow/block.ts

```diff
@@ -78,6 +82,7 @@ export const mutations: MutationTree<IFlowsState> = {
     Vue.delete(findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config!, key)
   },
   block_updateConfigByPath(state, {blockId, path, value}: {blockId: string, path: string, value: object | string}) {
+    // todo: this might still break reactivity
```

Comment:
Is the TODO still relevant or just a perso note during implementation?
Might be useful to track it somewhere in JIRA